.travis.yml

```diff
@@ -59,18 +60,18 @@ matrix:
       - pip install -r cirq/contrib/contrib-requirements.txt
       - pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/performance
-     
+
   - os: windows
     env: NAME=pytest (Windows)
     language: sh
     python: "3.7.3"
     before_install:
-    - choco install python --version 3.7.3
-    - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
-    - python -m pip install --upgrade pip wheel
+      - choco install python --version 3.7.3
+      - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
+      - python -m pip install --upgrade pip wheel
     install:
-    - python -m pip install -r requirements.txt
-    - python -m pip install -r cirq/contrib/contrib-requirements.txt
-    - python -m pip install -r dev_tools/conf/pip-list-dev-tools.txt
-    script: check/pytest --benchmark-skip
-    
+      - pip install -r requirements.txt
```

Comment:
Is this change actually necessary for the caching to work? In general we want to avoid saying just `pip` instead of `python -m pip` because the latter is far more reliable. Sometimes `python` refers to a version of python different than what `pip` is targeting.